api-reference/10 UI Components/dxSelectBox/1 Configuration/fieldTemplate.md

```diff
@@ -31,7 +31,7 @@ The example below shows how to define the **fieldTemplate**. If you want to cust
             fieldTemplate(data, container) {
                 container.append(
                     $('<div>').dxTextBox({
-                        text: data.Category + ": " + data.Name,
+                        value: data && data.Name,
```

Comment:
Why did you change the field's text?

Reply:
Please, refer to the T1156423 ticket

Reply:
It says nothing about why you ditched data.Category